unstract/connectors/src/unstract/connectors/filesystems/exceptions.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any
+
+
+class UnstractFSConnectorException(Exception):
+    """Base class for database-related exceptions from Unstract connectors."""
+
+    def __init__(self, *args: object, detail: Any, status_code: Any = 500) -> None:
+        super().__init__(*args)
+        self.detail = detail
+        self.status_code = status_code
```

Comment:
@kirtimanmishrazipstack please check if this works, try to propagate the same change to other similar overrides done to the constructor
```suggestion
    status_code = 500
```

Reply:
@chandrasekharan-zipstack this won't work. Also why do we need to change that? status_code is default 500

Reply:
My bad, just noticed that its inheriting `Exception` class alone